prow/config.yaml

```diff
@@ -18,6 +18,10 @@ triggers:
   - google/cadvisor
   trusted_org: kubernetes
 
+heart:
```

Comment:
Thinkins out loud, wouldn't it make sense to extend prow/plugins.yaml for plugin-specific config instead of this?